controllers/discovery/searchUser.js

```diff
@@ -87,6 +87,7 @@ const SearchUser = async (req, res) => {
           AND u.user_id != :userId
           AND u.is_anonymous = false
           AND u.is_banned = false
+          AND u.user_id != :admin_user_id
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change is logically correct as it excludes the admin user from search results. However, it's important to note that this approach relies on an environment variable `BETTER_ADMIN_ID` to identify the admin user. If there are multiple admins or if the admin ID changes, this code will need to be updated. A more robust solution would be to have a flag in the user model indicating whether a user is an admin or not and exclude based on that flag.
```diff
- AND u.user_id != :admin_user_id
+ AND u.is_admin = false
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->